src/networking/network.rs

```diff
@@ -346,6 +364,10 @@ impl Network {
                         warn!("SaitoMessage::BlockchainSavedBlock recv'ed by network");
                         Network::send_my_block_to_peers(block_hash).await;
                     }
+                    SaitoMessage::NetworkNewTransaction { transaction: tx } => {
```

Comment:
I think the intention which David has was for these message to indicate where they are coming from, so "NetworkNewTransaction" would imply that the message is coming from the SNDTRANS handler. We also need a way to handle transactions which we've created "ourself", which I'm now realizing is not exactly clear with saitocli... I guess this would really be from the wallet, e.g. SaitoMessage::WalletNewTranasction, so perhaps for now when we create tx via saitocli they should go through the wallet(even though we won't actually be using the wallet yet to get the inputs, we will in the future do something like this)